src/Profiles/BicyclePower/ANTPLUS_BicyclePowerPrivateDefines.h

```diff
@@ -0,0 +1,119 @@
+
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_PEDALPOWER_POWERONLY_BYTE 					2
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_CADENCE_POWERONLY_BYTE 						3
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_ACCUMULATEDPOWERLSB_POWERONLY_BYTE			4
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_ACCUMULATEDPOWERMSB_POWERONLY_BYTE			5
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_INSTANTPOWERLSB_POWERONLY_BYTE				6
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_INSTANTPOWERMSB_POWERONLY_BYTE				7
+
+/* Wheel Torque Main Data Page */
+
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_DATAPAGE_WHEELTORQUE_NUMBER						0x11 // New
+
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_WHEELTICKS_WHEELTORQUE_BYTE 					2 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_CADENCE_WHEELTORQUE_BYTE 						3 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_WHEELPERIODLSB_WHEELTORQUE_BYTE				4 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_WHEELPERIODMSB_WHEELTORQUE_BYTE				5 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_ACCUMULATEDTORQUELSB_WHEELTORQUE_BYTE			6 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_ACCUMULATEDTORQUEMSB_WHEELTORQUE_BYTE			7 // New
+
+/* Standard Crank Torque Main Data Page */
+
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_DATAPAGE_CRANKTORQUE_NUMBER						0x12 // New
+
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_CRANKTICKS_CRANKTORQUE_BYTE 					2 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_CADENCE_CRANKTORQUE_BYTE 						3 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_CRANKPERIODLSB_CRANKTORQUE_BYTE				4 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_CRANKPERIODMSB_CRANKTORQUE_BYTE				5 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_ACCUMULATEDTORQUELSB_CRANKTORQUE_BYTE			6 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_ACCUMULATEDTORQUEMSB_CRANKTORQUE_BYTE			7 // New
+
+/* Pedal Smoothness Main Data Page */
+
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_DATAPAGE_PEDALSMOOTHNESS_NUMBER						0x13 // New
+
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_LEFTEFFECTIVENESS_PEDALSMOOTHNESS_BYTE		2 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_RIGHTEFFECTIVENESS_PEDALSMOOTHNESS_BYTE		3 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_LEFTSMOOTHNESS_PEDALSMOOTHNESS_BYTE			4 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_RIGHTSMOOTHNESS_PEDALSMOOTHNESS_BYTE			5 // New
+
+/* Crank Torque Frequency Main Data Page
+
+NOTE: CTF messages are big endian. The byte order for multi-byte
+fields in these messages is reversed with respect to standard ANT+ messages
+
+Not implemented.
+*/
+
+/*
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_DATAPAGE_CTF_NUMBER						0x20 // New
+
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_SLOPEMSB_CTF_BYTE								2 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_SLOPELSB_CTF_BYTE								3 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_TIMESTAMPMSB_CTF_BYTE							4 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_TIMESTAMPLSB_CTF_BYTE							5 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_TORQUETICKSTAMPMSB_CTF_BYTE					6 // New
+#define ANTPLUS_BICYCLEPOWER_DATAPAGE_TORQUETICKSTAMPLSB_CTF_BYTE					7 // New
+*/
+
+/* Manufacturer Information */

```

Comment:
need to remove common datapages as they are already in shared code area, im sorry if you based this off HR example. HR doesn't use common datapages which could make this confusing.